src/types.ts

```diff
@@ -621,6 +641,44 @@ export interface UserComplianceResponseObject {
   reason: string | null,
 }
 
+export interface ProfilePrivateResponseObject {
```

Comment:
since this is a superset, could we have this extend the `ProfilePublicResponseObject`?

Reply:
genius